src/nylo/tokens/numstr.py

```diff
@@ -1,72 +1,97 @@
+# This file is a part of nylo
```

Comment:
numstr.py should not have the copyright. Only file beggining with __ shall have them.